services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -124,11 +124,29 @@ function checkUnicity(primaryKeys, uniqueIndexes, columnName) {
       indexColumnName.length === 1 && indexColumnName.includes(columnName));
 
   const isPrimary = _.isEqual([columnName], primaryKeys);
-  return { isPrimary, isUnique };
+  return isPrimary || isUnique;
```

Comment:
This is a little refacto to reduce cognitive complexity of `createAllReferences` function.